### ISSUE TITLE
bug fix displaying same image multiple times in markdown

### ISSUE THREAD

**Issue by hubschne** [bsorrentino/extract-pdf-images#7]
Hello again,

I ran into a bug and wanted to ask if you could fix it.

Unfortunately, images that are displayed on multiple slides only appear on the first instance in the markdown file for me. Here is an [Example_Presentation.pdf](https://github.com/bsorrentino/pdf-tools/files/11611188/Example_Presentation.pdf) and the [generated markdown file](https://github.com/bsorrentino/pdf-tools/files/11611197/Example_Presentation.md)
(the same image is displayed in the pdf slides on the last two slides, but in markdown it is only shown on the first instance slide).
I am using node v20.2.0 and your current pdf-tool version v1.1.0.
When I run the command `pdftools p2md`, I get the following output: 

```
{ pageseparator: '---' }


folder Example_Presentation doesn't exist, try to create
WARN: file Example_Presentation/Example_Presentation.fonts.json doesn't exists!
**image name g_d0_img_p3_3 not found!**
monospace or code font doesn't exists!
monospace or code font doesn't exists!
monospace or code font doesn't exists!
monospace or code font doesn't exists!
```

Thanks a lot in advance!

**Comment by bsorrentino** [comment]
Hi @hubschne thanks for feedback

I'll try to figure out how to solve it 😉

**Comment by bsorrentino** [comment]
solved by release `1.2.0`
